src/main/scala/firrtl/stage/FirrtlAnnotations.scala

```diff
@@ -290,24 +290,3 @@ case object PrettyNoExprInlining extends NoTargetAnnotation with FirrtlOption wi
   * @param op the op that should never be folded
   */
 case class DisableFold(op: ir.PrimOp) extends NoTargetAnnotation with FirrtlOption
-
-object DisableFold extends HasShellOptions {
-
-  private val mapping: Map[String, ir.PrimOp] = PrimOps.builtinPrimOps.map { case op => op.toString -> op }.toMap
-
-  override val options = Seq(
-    new ShellOption[String](
-      longOption = "dont-fold",
-      toAnnotationSeq = a => {
-        mapping
-          .get(a)
-          .orElse(throw new OptionsException(s"Unknown primop '$a'. (Did you misspell it?)"))
-          .map(DisableFold(_))
-          .toSeq
-      },
-      helpText = "Disable folding of specific primitive operations",
-      helpValueName = Some("<primop>")
-    )
-  )
-
-}
```

Comment:
I think it breaks binary compatibility, but this class should belong to FIRRTL privately, and won't affect users experience.

Reply:
Somehow I seem to have lost my comment about this. 

I'd like to get this into 1.4.3. Annoyingly, this `DisableFold` thing was already in 1.4.2... I agree that this is entirely irrelevant to user experience. However, I'm concerned about breaking binary compatibility here.

What about leaving this in and deprecating it?